packages/app/README.md

```diff
@@ -66,6 +66,17 @@ visualization, which is useful when H5Web is embeded inside another app.
 <App startFullscreen />
 ```
 
+### `initialPath?: string` (optional)
+
+The path to select within the file when the app is first rendered. Defaults to
+`'/'`.
+
+```tsx
+<MockProvider>
+  <App initialPath="/nD_datasets/threeD" />
+</MockProvider>
```

Comment:
It seems strange to pass this to `App` rather than through the `Provider` that is supposed to handle the files.

We should perhaps add the `initialPath` to the provider context and get it back in the `App`.

This being said, I have no scruple to merge this as-is as this is not the main focus of the PR.

Reply:
`initialPath` is just a UI state, and the `App` is the UI, so it still makes sense IMO. That being said, I get that there's coupling between the path and the file that is passed to the provider, but since the `App` and its provider are always rendered in the same place, I don't reckon there'd be a real gain in making it more convoluted by passing the path through the provider's context.